briskedu/brisk-learning
# Improve error handling when services are down

Summary:
Improving error handling when services don't respond (like right now where API keys are expired)

Test Plan:
1. Confirm generating error shows message
2. Confirm I can close error message
3. Confirm I can retry
